redbot/core/modlog.py

```diff
@@ -940,16 +940,24 @@ async def create_case(
         The last known username of the user
         Note: This is ignored if a Member or User object is provided
         in the user field
+
+    Raises
+    ------
+    ValueError
+        If the action type is not a valid action type.
+    RuntimeError
+        If user is the bot itself
     """
     case_type = await get_casetype(action_type, guild)
     if case_type is None:
-        return
+        raise ValueError("That is not a valid action type.")
 
     if not await case_type.is_enabled():
         return
 
-    if user == bot.user:
-        return
+    user_id = user if type(user) == int else user.id
```

Comment:
We could get away with checking against `type()` if it weren't a user-provided argument but since it isn't, we should use `isinstance` here instead:
```suggestion
    user_id = user if isinstance(user, int) else user.id
```